setup.py

```diff
@@ -47,7 +47,7 @@
 install_requires = [
     'boto3>=1.9.83',
     'invenio-files-rest>=1.0.0a23',
-    's3fs>=0.1.5',
+    's3fs>=0.1.5,<0.3.0', # Newer versions only allow python >= 3.5
```

Comment:
We can remove python 2 support if you like. Since this module won't be in production until after January 2020.

Reply:
There is nothing I would love more than removing python 2.7 support, but I know at least one production system using it, I'll check with them what python version they are using and fix this accordingly